shopify-cli.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |spec|
     and webhooks.
   HERE
   spec.homepage = "https://shopify.github.io/shopify-app-cli/"
-  spec.required_ruby_version = Gem::Requirement.new(">= 2.3.0")
+  spec.required_ruby_version = Gem::Requirement.new(">= 2.5.0")
```

Comment:
I'm not sure of the significance but https://shopify.github.io/shopify-app-cli/getting-started/ says 2.5.1.

Reply:
If it's all the same, maybe we should depend on `2.5` instead?

Reply:
So, make it `">= 2.5"`?